dali/operators/reader/loader/webdataset_loader.cc

```diff
@@ -255,12 +257,13 @@ void WebdatasetLoader::ReadSample(vector<Tensor<CPUBackend>>& sample) {
     current_wds_shard->Seek(component.offset);
 
     // Skipping cached samples
-    const std::string source_info =
+    const std::string image_key =
         make_string_delim(" ", "Archive:", paths_[current_sample.wds_shard_index],
                     GetSampleSource(current_sample), "; Component offset:", component.offset);
     DALIMeta meta;
-    meta.SetSourceInfo(source_info);
-    if (ShouldSkipImage(source_info)) {
+    meta.SetSourceInfo(make_string_delim(':', paths_[current_sample.wds_shard_index],
```

Comment:
At some point we should document each reader's format for source_info. I guess this could be done on a separate PR for all readers.